scss/calorie-counter-responsive.scss

```diff
@@ -0,0 +1,31 @@
+@import './mixins';
+
+@include md {
+  body {
+    font-size: 20px;
+    background-color: red;
+  }
+
+  .container {
+    max-width: 800px;
+  }
+
+  .calorie-counter__button {
+    padding: 8px 12px;
+  }
+}
+
+@include lg {
+  body {
+    font-size: 22px;
```

Comment:
It's not a good practice to target global elements like the body form your component's code.